backend/app/settings.py

```diff
@@ -20,6 +20,8 @@ class Config(object):
     TEST_DIR = os.path.abspath(os.path.join(PROJECT_ROOT, "tests"))
     SQLALCHEMY_TRACK_MODIFICATIONS = False
     SUBGRAPH_ENDPOINT = os.getenv("SUBGRAPH_ENDPOINT")
+    SUBGRAPH_TIMEOUT = int(os.getenv("SUBGRAPH_TIMEOUT", 10))  # seconds
+    # SUBGRAPH_TIMEOUT value of 0 means no timeout. Fallback to `requests` defaults.
```

Comment:
i think this comment is not needed, it can be deduced of code implemnetation

Reply:
+1